src/components/AlertMessage.tsx

```diff
@@ -40,9 +40,6 @@ const AlertContainer = styled.div<StyleProps>`
       case AlertType.Info: {
         return css`
           --alert-accent-color: var(--color-text-1);
-
-          --alert-default-background-opacity: 0.133; // Relative
```

Comment:
default background opacity is already set to 0.1 (for all other variants) - this seems unnecessary, verified it looks fine in all modes

| Classic | Light | Dark |
| ---- | ---- | ---- |
| ![Screenshot 2024-02-02 at 3 19 57 PM](https://github.com/dydxprotocol/v4-web/assets/70078372/a661a85c-115a-4fa0-a450-bb7c7992d403) | ![Screenshot 2024-02-02 at 3 20 01 PM](https://github.com/dydxprotocol/v4-web/assets/70078372/53af0319-eb0d-440c-b473-c760001c94ab) | ![Screenshot 2024-02-02 at 3 20 05 PM](https://github.com/dydxprotocol/v4-web/assets/70078372/2dbaf724-04ad-4968-8d24-bc3a4dff1dd0) |